packages/widgetbook_core/lib/src/themes/themes.dart

```diff
@@ -43,7 +43,7 @@ class Themes {
     secondary: Color(0xFFBBC7DB),
     onSecondary: Color(0xFF263141),
     secondaryContainer: Color(0xFF3C4858),
-    onSecondaryContainer: Color(0xFFD7E3F8),
+    onSecondaryContainer: Color(0xff36485F),
```

Comment:
Why was this changed?

Reply:
This is related to this:
https://github.com/widgetbook/widgetbook/pull/415#discussion_r1058173179

And this value now matches the Figma design.
I think in general we should work on the themes such that all colors match the new design.